public/static/js/tour-controller.js

```diff
@@ -7,9 +7,16 @@ function b64DecodeUnicode(str) {
     }).join(''));
 }
 
+dlangTourApp.config(['$locationProvider', function($locationProvider) {
+	$locationProvider.html5Mode({
+		enabled: true,
+		requireBase: false
+	});
```

Comment:
This is needed for `$location.search()` to work (otherwise Angular will try to use postfix hashes)